APMrover2/control_modes.cpp

```diff
@@ -1,5 +1,7 @@
 #include "Rover.h"
 
+static const int16_t CH_7_PWM_TRIGGER = 1800;
+
 void Rover::read_control_switch()
```

Comment:
This uses up 2 more bytes of RAM that we are running out of - not so much in Rover I know but . . .
Why change this?